app/Http/Controllers/ArticlesController.php

```diff
@@ -4,50 +4,62 @@
 
 use Illuminate\Http\Request;
 use App\Models\Article;
+use App\Models\User;
 use App\Http\Requests\ArticleSave;
 
 class ArticlesController extends Controller
 {
-    public function index()
-    {
-        $articles = Article::whereNotNull('published_at');
-
-        return $articles;
-    }
-
+    /**
+     * Show the post creation page
+     *
+     * @return \Illuminate\Http\Response
+     */
     public function create()
     {
-        \Auth::user()->user_type != 2 ? abort(403) : '';
+        \Auth::user()->mustBe([User::AUTHOR]);
```

Comment:
Terminate this checking using  customised [middleware](https://laravel.com/docs/5.4/middleware).